regression-test/suites/query_p0/system/test_table_options.groovy

```diff
@@ -185,5 +178,32 @@ suite("test_table_options") {
     """
 
     qt_select """select * from information_schema.table_options where table_schema=\"${dbName}\" order by TABLE_NAME; """
-    sql "drop database if exists ${dbName}"
+    sql """
+        drop table test_row_column_page_size2;
+    """    
+    qt_select """select * from information_schema.table_options where table_schema=\"${dbName}\" order by TABLE_NAME; """    
+
+    def user = "table_options_user"
+    sql "DROP USER IF EXISTS ${user}"
+    sql "CREATE USER ${user} IDENTIFIED BY '123abc!@#'"
+    sql "GRANT SELECT_PRIV ON information_schema.table_properties  TO ${user}"
+
+    def tokens = context.config.jdbcUrl.split('/')
+    def url=tokens[0] + "//" + tokens[2] + "/" + "information_schema" + "?"
```

Comment:
connect(user = 'test_wg_priv_user2', password = '', url = context.config.jdbcUrl) {
    qt_select "xxx"
}

Reply:
This I tried first use jdbcUrl of test case , since user dont have any privileges, it failed to connect to the database of test case.
so I gave a minimal permission for information_schema, used it to connect .